### PR TITLE
Update the nomination process in Governance

### PR DESCRIPTION
**Pull request by luzizhuo**
Signed-off-by: luzizhuo <496521310@qq.com>

### What problem does this PR solve?

1. Update the way of member nomination to PR instead, because it's not so efficient at the community meeting.
2. Propose to add the nominee's contribution and the reason in the nomination PR
3. Add description of how the nomination is approved

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.2
  - [ ] release-2.1

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Review by STRRL — COMMENTED** [review]
/cc @chaos-mesh/maintainers 

Cool!

File: GOVERNANCE.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To become a Chaos Mesh member, you are expected to:
     - Join community discussions (e.g. community meetings, Slack).
 - Sponsored by at least 1 Chaos Mesh [maintainer or committer](https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md).
 
-Contributors that meet the above requirements will then be nominated by their sponsor at the next available [community meeting](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit) to become a member. The maintainers will confer and decide whether to grant membership or not. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
+Contributors that meet the above requirements will then be nominated by their sponsor through a PR. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. The maintainers will confer and decide whether to grant membership or not. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
```
Comment (dcalvin) [inline GOVERNANCE.md:34]:
```suggestion
Contributors that meet the above requirements will then be nominated by their sponsor through a PR. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. To get the PR merged, at least two approvals are required from committers or maintainers. Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.
```

**Review by dcalvin — COMMENTED** [review]
Only one suggested change on the approval process of Members. The rest LGTM

File: GOVERNANCE.md (hunk 1 above)
Comment (luzizhuo) [reply GOVERNANCE.md:34]: OK. I will update the corresponding statements in the form below.

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**